mne/preprocessing/eog.py

```diff
@@ -132,23 +131,26 @@ def _find_eog_events(eog, event_id, l_freq, h_freq, sampling_rate, first_samp,
 
 
 def _get_eog_channel_index(ch_name, inst):
-    """Get EOG channel index."""
+    """Get EOG channel indices."""
+    # Ensure we have a list of channel names
     if isinstance(ch_name, str):
-        # Check if multiple EOG Channels
-        if ',' in ch_name:
+        if ',' in ch_name:  # backward-compat
```

Comment:
shall we deprecate this ","  option. I would say so.

Reply:
+1 for deprecation